bin/fs-default-project-config.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+var fs = require('fs');
+var path = require('path');
+
+var lib = require('../lib/index.js');
+var resourcesFolder = path.join(__dirname, '../resources');
+var targetFiles = fs.readdirSync(resourcesFolder);
+
+console.log('----------------------------');
+console.log('---Deploying config files---');
+console.log('----------------------------');
+console.log(
+    'Copy Completed: ',
+    lib.copyConfig(
+        targetFiles.map(function(file) {
+            return path.join(resourcesFolder, file);
+        }),
+        path.join(process.cwd())
+    )
+);
```

Comment:
Can we investigate something like;

```
var npm = require("npm");
npm.commands.install(['hello-world@0.0.1']);
```

Or the quick win

```
var exec = require('child_process').execSync;
exec('npm install grunt-eslint@1.0.0 --save-dev');
```

At the end of copying all the files?

We're also probably best doing a quick check at the start of this to;

```
try {
  var packageDetails = require(process.cwd() + '/package.json);
  if (packageDetails.name === 'fs-default-project-config') {
    console.error('Run from the wrong directory!');
    exit 1;
  }
catch (e) {
  console.error('No existing project found in this folder. If you wish to update a project, please npm init first.');
  exit 1;
}
```

Or something like that :D